app/routes/core.rb

```diff
@@ -102,8 +102,26 @@ def tracks
         end
 
         def css_url
-          @css_url || "/css/application.css?t=#{File.mtime('./public/css/application.css').to_i}"
+          @css_url ||= css_url_generate('/css/application.css')
```

Comment:
Also fixes the bug which meant that @css_url was never cached.
I assume that was not intentional.